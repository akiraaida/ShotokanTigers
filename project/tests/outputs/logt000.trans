10 John Doe             00000 00000.00 S 
00                      00000 00000.00   
10 admin                00000 00000.00 A 
00                      00000 00000.00   
