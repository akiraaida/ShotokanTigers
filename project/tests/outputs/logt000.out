Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
Please enter your session type: 
Please enter a command: 
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
Please enter a command: 
ERROR, YOU HAVE NOT LOGGED IN YET.
